physical/etcd.go

```diff
@@ -79,18 +82,54 @@ func newEtcdBackend(conf map[string]string) (Backend, error) {
 	if address, ok := conf["address"]; ok {
 		machines = address
 	}
+	machinesParsed := strings.Split(machines, EtcdMachineDelimiter)
```

Comment:
Is there any way that we can verify that these machines are correct? e.g. if they are URLs, can you run them through `url.Parse`; if they are IP addresses can you run them through `net.ParseIP`?